Makefile

```diff
@@ -174,6 +175,16 @@ dev/syncdeps:
 #
 # Release targets
 #
+
+.PHONY: release/versionbump
+release/versionbump:
+	$(MAKE) CHANNEL=stable IS_DEFAULT_CHANNEL=1 release/update-pkg-manifest && \
```

Comment:
@alinadonisa correct me if I am wrong but I think we don’t release to stable right away.

Reply:
> @alinadonisa correct me if I am wrong but I think we don’t release to stable right away.

@rail, @Chris we will release on beta channel first and after the feedback  we go for the stable channel